lib/content-services/src/lib/search/components/search-filter/search-filter.component.html

```diff
@@ -3,8 +3,7 @@
     <button *ngIf="displayResetButton && facetFiltersService.responseFacets"
             mat-button
             color="primary"
-            matTooltip="{{ 'SEARCH.FILTER.BUTTONS.RESET-ALL.TOOLTIP' | translate }}"
-            matTooltipPosition="right"
```

Comment:
Same here

Reply:
yes, added some screenshots

Reply:
should be working fine, added a few screenshots from the ACA smoke tests